symphony/bdk/core/auth/auth_session.py

```diff
@@ -53,3 +53,42 @@ def key_manager_token(self, value):
         :param value: the new key manager token.
         """
         self._key_manager_token = value
+
+
+class OboAuthSession(AuthSession):
+    """RSA OBO Authentication session handle to get the OBO session token from.
+    It uses a OboAuthenticator to actually retrieve the tokens when needed.
+    """
+
+    def __init__(self, authenticator, user_id: int = None, username: str = None):
+        """
+        At least user_id or username should be defined.
```

Comment:
I guess both should not be specified.

Reply:
I didn't find the best way to initialise the auth_session here because we cannot have the overloaded method in Python.

Reply:
then should we throw an exception if both or none are passed?